src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -0,0 +1,9 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class ColorSupplier {
+    public String getRandomColor() {
+        return String.valueOf(Color.values() [new Random().nextInt(Color.values().length)]);
```

Comment:
```suggestion
        return Color.values() [new Random().nextInt(Color.values().length)].name();
```

Reply:
also let's make the Random object a field of the class so that it is not created every time